query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverterConfig.java

```diff
@@ -30,9 +38,24 @@ public PinotFunctionConverterConfig(Config config) {
     } else {
       this.distinctCountAggregationFunction = DEFAULT_DISTINCT_COUNT_AGGREGATION_FUNCTION;
     }
+    if (config.hasPath(DISTINCT_COUNT_AGGREGATION_OVERRIDES)) {
+      Config overridesConfig = config.getConfig(DISTINCT_COUNT_AGGREGATION_OVERRIDES);
+      this.distinctCountAggOverrides =
+          overridesConfig.entrySet().stream()
```

Comment:
nit: keySet, not using the whole entry

Reply:
this is config object - no keySet available, only entrySet..

Reply:
ah - `ConfigObject` has keySet (it implements map), `Config` does not. So `.root().keySet()` would work. FWIW, the two are equivalent for a one level config but I would say keySet version better aligns with your intention - getting the immediate keys, rather than what entrySet does (recurses into all children and produces path expressions mapped to any descendent value).